torch/onnx/experimental/fonnx/v1/custom_operators.py

```diff
@@ -0,0 +1,63 @@
+from contextlib import contextmanager
+
+
+def register_custom_operator(
+    domain: str,
+    operator: str,
+    opset_version: int,
```

Comment:
nit: Wonder if this should be part of `symbolic_fn` now. For OnnxFunction I think yes.

Reply:
would the registry have to `hack` into `symbolic_fn` to learn "which opset" that operator belongs too? The current FX-exporter PoC doesn't have the concept of opset, so let's check whether the current approach is applicable

Reply:
May want to keep `opset_version` for now per offline discussion.

This needs discussion/clarification from onnxscript/atenlib side.